### PR TITLE
8196018: java/awt/Scrollbar/ScrollbarMouseWheelTest/ScrollbarMouseWheelTest.java fails

### PR DESCRIPTION
**Pull request by azvegint**
The issue contains evaluation:
* the test is not failing on Windows
* it also fails on Linux(but provided stack trace with failure is from different test)

I rechecked the test and it does not fail on all platforms.

Test is stabilized:
* replaced synchronization on windowActivated event with standard `robot.waitForIdle(); robot.delay(1000);`
* Robot autoDelay decreased to 250 to make the test go faster. It it still a reasonable value.

CI testing looks good also.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8196018](https://bugs.openjdk.org/browse/JDK-8196018): java/awt/Scrollbar/ScrollbarMouseWheelTest/ScrollbarMouseWheelTest.java fails

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**) ⚠️ Review applies to [c2ada26e](https://git.openjdk.org/jdk/pull/11213/files/c2ada26e4200121b339f0f67a5df2da31604897c)
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**)
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Author)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11213/head:pull/11213` \
`$ git checkout pull/11213`

Update a local copy of the PR: \
`$ git checkout pull/11213` \
`$ git pull https://git.openjdk.org/jdk pull/11213/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11213`

View PR using the GUI difftool: \
`$ git pr show -t 11213`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11213.diff">https://git.openjdk.org/jdk/pull/11213.diff</a>

</details>

### CODE REVIEW

**Review by prsadhuk — APPROVED** [review]

File: test/jdk/java/awt/Scrollbar/ScrollbarMouseWheelTest/ScrollbarMouseWheelTest.java [hunk 1]
```diff
@@ -108,7 +107,9 @@ public void test() {
         testComp(sb2, true);
         // down on sb2
         testComp(sb2, false);
+
         frame.dispose();
+
         System.out.println("Test done.");
         if (sb1upevents == REPS &&
```
Comment (mrserb) [inline test/jdk/java/awt/Scrollbar/ScrollbarMouseWheelTest/ScrollbarMouseWheelTest.java:114]: Note that all these flags are updated on one thread and used on another.

Comment (azvegint) [reply test/jdk/java/awt/Scrollbar/ScrollbarMouseWheelTest/ScrollbarMouseWheelTest.java:116]: Updated, also added extra delay after test.

**Review by mrserb — APPROVED** [review]

**Review by DamonGuy — APPROVED** [review]
Tested locally on macOS and the test works normally with the test removed from the problem list.

**Pull request closed without merging**